client/layout/guided-tours/docs/examples/tours/simple-payments-end-of-year-guide.js

```diff
@@ -1,9 +1,10 @@
 /**
  * External dependencies
+ *
+ * @format
```

Comment:
this is strange. I wouldn't have expected this to be happening anymore since we should always be formatting files as we add them

Reply:
This is in a `docs/examples` directory -- maybe it was skipped during the mass-reformat?